src/config/config-utils.js

```diff
@@ -21,14 +25,87 @@ import pkgJson from '../package.cjs';
  */
 export async function validateDotEnv(dir = process.cwd()) {
   if (await GitUtils.isIgnored(dir, '.env')) {
-    return;
+    return true;
   }
   process.stdout.write(chalk`
 {yellowBright Warning:} Your {cyan '.env'} file is currently not ignored by git. 
 This is typically not good because it might contain secrets 
 which should never be stored in the git repository.
+`);
+  return false;
+}
+
+const hlxFolder = '.hlx';
+const tokenFileName = '.hlx-token';
+const tokenFilePath = path.join(hlxFolder, tokenFileName);
```

Comment:
```suggestion
const tokenFilePath = path.resolve(hlxFolder, tokenFileName);
```

Reply:
Unfortunately, this doesn't work, because `GitUtils.isIgnored` wants a separate relative path from the git project.

https://github.com/adobe/helix-cli/pull/2471/files#diff-d4deb0baa7399a2b0cb5f027dc174ec9dbfc4d9c5df876bc29f6f1503cf7aa6eR79